python/paddle/fluid/tests/unittests/test_complex_sum_layer.py

```diff
@@ -37,6 +38,21 @@ def test_complex_x(self):
                 target = np.sum(input, axis=(1, 2))
                 self.assertTrue(np.allclose(result, target))
 
+    def test_complex_basic_api(self):
+        input = rand([2, 10, 10]).astype(self._dtype) + 1j * rand(
```

Comment:
make sure we test cases include `[float32, float64] * [CPU, GPU]`, the original unittest for complex is incomplete

Reply:
done, thanks.